docs/photoniq/prerendering/validate-prerendering.md

```diff
@@ -7,14 +7,14 @@ In order to confirm Dynamic Prerendering is running, you can check the the healt
 
 ## Check Dynamic Prerendering Health
 
-To see if Dynamic Prerendering is currently running, you can use the [Service Health Status](prerendering-tasks-api.md#service-health-status) API endpoint.
+To see if Dynamic Prerendering is currently running, you can use the [Gets service health status](/docs/apiPrerendering#/paths/api-prerender-v1-health/get) API endpoint.
 
 ## View Prerendered Pages
 
 Here are ways to view the prerendered pages:
 
 1. **Use Developer Tools**: Many modern browsers have developer tools that allow you to emulate different user agents, including web crawlers. Switch to a crawler user agent and reload the page to see the prerendered version.
 
-2. **API Endpoints**: Use the [Fetch Prerendered Page Content](prerendering-tasks-api#fetch-prerendered-page-content) endpoint to view the prerendered page.
+2. **API Endpoints**: Use the [Fetches the prerendered page content](/docs/apiPrerendering#/paths/api-prerender-v1-render-mobile---url/get) endpoint to view the prerendered page.
```

Comment:
Does this link work?

Reply:
Yes, all links working.

Reply:
You can check the links here:
https://docs-c0sw505ni-team-macrometa.vercel.app/docs/photoniq/prerendering/get-started-prerendering
https://docs-c0sw505ni-team-macrometa.vercel.app/docs/photoniq/prerendering/validate-prerendering